### PR TITLE
Remove kolla_build_customizations_rocky defaults in Zed.

### PR DESCRIPTION
**Pull request by assumptionsandg**

### CODE REVIEW

File: etc/kayobe/environments/ci-builder/stackhpc-ci.yml [hunk 1]
```diff
@@ -89,3 +89,6 @@ stackhpc_docker_registry_password: !vault |
 # Username and password of the overcloud host image repository.
 stackhpc_image_repository_username: "{{ stackhpc_docker_registry_username }}"
 stackhpc_image_repository_password: "{{ stackhpc_docker_registry_password }}"
+
+# SELinux set to permissive mode.
+selinux_state: permissive
```
Comment (Alex-Welsh) [inline etc/kayobe/environments/ci-builder/stackhpc-ci.yml]: set to disabled. If you try to set to permissive, it'll try to reboot the machine and error out

File: etc/kayobe/kolla.yml [hunk 2]
```diff
@@ -277,47 +277,7 @@ kolla_build_customizations_common:
...
-    - ovn22.09-central
-  ovn_sb_db_server_packages_override:
-    - ovn22.09-central
-  openvswitch_base_packages_remove:
-    - openvswitch
-    - python3-openvswitch
-  openvswitch_base_packages_append:
-    - openvswitch2.17
-    - python3-openvswitch2.17
+kolla_build_customizations_rocky: {} # TODO: Check whether this is actually correct. It seems that it was previously being skipped for rocky
```
Comment (markgoddard) [inline etc/kayobe/kolla.yml]: Comment can be removed.

File: etc/kayobe/environments/ci-builder/stackhpc-ci.yml [hunk 3]
```diff
@@ -89,3 +89,6 @@ stackhpc_docker_registry_password: !vault |
 # Username and password of the overcloud host image repository.
 stackhpc_image_repository_username: "{{ stackhpc_docker_registry_username }}"
 stackhpc_image_repository_password: "{{ stackhpc_docker_registry_password }}"
+
+# SELinux set to disabled mode.
+selinux_state: disabled
```
Comment (markgoddard) [inline etc/kayobe/environments/ci-builder/stackhpc-ci.yml]: This needs to go in `etc/kayobe/environments/ci-builder/kolla/globals.yml`. I'm adding it in https://github.com/stackhpc/stackhpc-kayobe-config/pull/626 so you could remove if you like.

Comment (holliefae) [reply etc/kayobe/environments/ci-builder/stackhpc-ci.yml]: Done

**Comment by markgoddard** [comment]
LGTM. Ready for review?

**Comment by assumptionsandg** [comment]
> LGTM. Ready for review?

Going to run some tests first, once they pass I'll open it.

**Comment by markgoddard** [comment]
Incorporated this into https://github.com/stackhpc/stackhpc-kayobe-config/pull/654/

**Pull request closed without merging**